python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -124,13 +129,131 @@ def setDefaults(self):
+\code
+assembleCoadd.py --help
+\endcode
+To demonstrate usage of the AssembleCoaddTask in the larger context of multi-band processing, we will generate 
+the HSC-I & -R band coadds from HSC engineering test data provided in the ci-hsc package. To begin, assuming 
+that the lsst stack has been already set up, we must set up the obs_subaru and ci_hsc packages. 
+This defines the environment variable $CI_HSC_DIR and points at the location of the package. The raw HSC 
+data live in the $CI_HSC_DIR/raw directory. To begin assembling the coadds, we must first 
+<DL>
+  <DT>processCcd</DT>
+  <DD> process the individual ccds in $CI_HSC_RAW to produce calibrated exposures</DD>
+  <DT>makeSkyMap</DT>
+  <DD> create a skymap that covers the area of the sky present in the raw exposures</DD>
+  <DT>makeCoaddTempExp</DT>
+  <DD> warp the individual calibrated exposures to the tangent plane of the coadd</DD>
+</DL>
+We can perform all of these steps by running
+\code
+$CI_HSC_DIR scons warp-903986 warp-904014 warp-903990 warp-904010 warp-903988
+\endcode
+This will produce warped coaddTempExps for each visit. To coadd the wraped data, we call assembleCoadd.py as 
+follows:
+\code
+assembleCoadd.py --legacyCoadd $CI_HSC_DIR/DATA --id patch=5,4 tract=0 filter=HSC-I --selectId visit=903986 ccd=16 --selectId visit=903986 ccd=22 --selectId visit=903986 ccd=23 --selectId visit=903986 ccd=100 --selectId visit=904014 ccd=1 --selectId visit=904014 ccd=6 --selectId visit=904014 ccd=12 --selectId visit=903990 ccd=18 --selectId visit=903990 ccd=25 --selectId visit=904010 ccd=4 --selectId visit=904010 ccd=10 --selectId visit=904010 ccd=100 --selectId visit=903988 ccd=16 --selectId visit=903988 ccd=17 --selectId visit=903988 ccd=23 --selectId visit=903988 ccd=24\endcode
+that will process the HSC-I band data. The results are written in $CI_HSC_DIR/DATA/deepCoadd-results/HSC-I 
+You may also choose to run:
+\code
+scons warp-903334 warp-903336 warp-903338 warp-903342 warp-903344 warp-903346
+assembleCoadd.py --legacyCoadd $CI_HSC_DIR/DATA --id patch=5,4 tract=0 filter=HSC-R --selectId visit=903334 ccd=16 --selectId visit=903334 ccd=22 --selectId visit=903334 ccd=23 --selectId visit=903334 ccd=100 --selectId visit=903336 ccd=17 --selectId visit=903336 ccd=24 --selectId visit=903338 ccd=18 --selectId visit=903338 ccd=25 --selectId visit=903342 ccd=4 --selectId visit=903342 ccd=10 --selectId visit=903342 ccd=100 --selectId visit=903344 ccd=0 --selectId visit=903344 ccd=5 --selectId visit=903344 ccd=11 --selectId visit=903346 ccd=1 --selectId visit=903346 ccd=6 --selectId visit=903346 ccd=12
+\endcode
+to generate the coadd for the HSC-R band if you are interested in following multiBand Coadd processing as 
+discussed in \ref pipeTasks_multiBand (but note that normally, one would use the 
+\ref SafeClipAssembleCoaddTask_ "SafeClipAssembleCoaddTask" rather than AssembleCoaddTask to make the coadd.
```

Comment:
I'm not sure you want to frame this in the context of multi-band processing and ci_hsc.  These both seem like more of a tutorial than actually documenting the code itself.